back-end/routes/main-route.js

```diff
@@ -1,30 +1,46 @@
-const express = require('express');
+const express = require("express");
 const mainRouter = express.Router();
-const { addProduct, getproducts, deleteProduct, updateProduct, addStore, updateStore, getStores, deleteStore,
-  createItem, deleteItem, createOrder, getItems, getOrders } = require("../controllers/main-controller")
-const { register, getAllUsers, login } = require("../controllers/users_controller")
-
-mainRouter.post('/product', addProduct);
-mainRouter.get('/product', getproducts);
-mainRouter.delete('/product', deleteProduct);
-mainRouter.put('/product', updateProduct);
-
-mainRouter.post('/store', addStore);
-mainRouter.put('/store', updateStore);
-mainRouter.get('/store', getStores);
-mainRouter.delete('/store', deleteStore);
-
-mainRouter.post('/order', createOrder);
-mainRouter.get('/order', getOrders);
-mainRouter.delete('/order', deleteStore);
-
-mainRouter.post('/item', createItem);
-mainRouter.get('/item', getItems);
-mainRouter.delete('/item', deleteItem);
-
-mainRouter.post('/register', register);
-mainRouter.post('/login', login);
-mainRouter.get('/users', getAllUsers);
+const {
+  addProduct,
+  getproducts,
+  deleteProduct,
+  updateProduct,
+  addStore,
+  updateStore,
+  getStores,
+  deleteStore,
+  createItem,
+  deleteItem,
+  createOrder,
+  getItems,
+  getOrders,
+  getAllStores,
+  specificStores
+} = require("../controllers/main-controller");
+const {
+  register,
+  getAllUsers,
+  login,
+} = require("../controllers/users_controller");
 
+mainRouter.post("/product", addProduct);
+mainRouter.post('/getproduct', getproducts);
```

Comment:
How come this is a post and it calls `getproducts`?  `getproducts` should be` getProducts`. 
What does `getproducts` do exactly if it is getting the products then the path and HTTP method should be different